### PR TITLE
A configurable infobox

### PR DESCRIPTION
**Pull request by genofire**
Current there are the bars of
* memory usage
* loadavg
configurable

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -27,9 +27,13 @@
   "dataPath": "https://regensburg.freifunk.net/data/",
   "reverseGeocodingApi": "https://nominatim.openstreetmap.org/reverse",
   "siteName": "Freifunk Regensburg",
-  "showContact": false,
   "maxAge": 14,
   "nodeZoom": 18,
+  "nodeInfobox":{
```
Comment (xf-) [inline config.json]: space missing

File: README.md [hunk 2]
```diff
@@ -103,6 +99,20 @@ nodes being offline for more than than maxAge days are considered "lost".
 Max level to be applied by clicking a node or open a node. With value `18` near by buildings and streets should be visible.
 Interesting if one of configured map provider has zoom-level under `18`.
 
+## nodeInfobox
+
+### showContact (bool)
```
Comment (xf-) [inline README.md]: h3 is to large

Maybe current README needs more consistent between h4 or code syntax.
https://github.com/ffrgb/meshviewer#maplayers-list
https://github.com/ffrgb/meshviewer#fixedcenter-array-optional

File: config.json [hunk 3]
```diff
@@ -27,9 +27,13 @@
...
+    "showContact": true,
+    "hideMemory": false,
```
Comment (xf-) [inline config.json]: Should we split setting bout current hardware usage? Both are more or less bind together.

File: config.json [hunk 4]
```diff
@@ -27,9 +27,13 @@
   "dataPath": "https://regensburg.freifunk.net/data/",
   "reverseGeocodingApi": "https://nominatim.openstreetmap.org/reverse",
   "siteName": "Freifunk Regensburg",
-  "showContact": false,
   "maxAge": 14,
   "nodeZoom": 18,
+  "nodeInfobox":{
+    "showContact": true,
```
Comment (xf-) [inline config.json]: Please don't change the default value.

**Review by xf- — CHANGES_REQUESTED** [review]

**Comment by xf-** [comment]
Please use a prefix like [TASK] in commit message

**Comment by genofire** [comment]
Done

**Review by xf- — APPROVED** [review]

**Comment by xf-** [comment]
https://github.com/ffrgb/meshviewer/commit/ecf8fe7ec4145a161e0245cd4a1307109cb52b07 pushed

**Pull request closed without merging**